pkg/log/log.go

```diff
@@ -0,0 +1,20 @@
+package log
+
+import (
+	"os"
+
+	"github.com/Sirupsen/logrus"
+)
+
+var Log = logrus.New()
```

Comment:
Let's create a `log.New()` function instead to initialize a logger. Specific packages could then tweak their logger without modifying another goroutine's logger (as would be the case by sharing a global variable).